### DISCUSSION TITLE
Error: A local file has the same name as another local file.

### DISCUSSION THREAD

**Discussion by wtrdk** [abraunegg/onedrive#1932] [General]
After the first succesfull run of uploading a lot of pictures to OneDrive, on the second run, after adding some new pictures to the source location, I get a lot of these errors:
```
ERROR: A local file has the same name as another local file.
ERROR: To resolve, rename this local file: </path/filename>
Skipping uploading this new file: </path/filename>
```

Does anyone know what is going wrong? Do i really have to rename hundreds of pictures??

**Comment by abraunegg** [comment]
@wtrdk 
Microsoft OneDrive does not support the following - as an example:

* IMG_1234.JPG
* img_1234.jpg

On Linux they are separate files, to the OneDrive API they are the same name ... even though they are different files and possibly different content.

Your only option is to rename the files. See:

https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#local-file-and-folder-naming-conventions
